code/FulpstationCode/fulp_mobs/apostle.dm

```diff
@@ -0,0 +1,184 @@
+/mob/living/simple_animal/hostile/megafauna/apostle
+	name = "apostle"
+	desc = "The heavens' wrath. You might've fucked up real bad to summon one."
+	health = 3500
+	maxHealth = 3500
+	attack_verb_continuous = "purges"
+	attack_verb_simple = "purge"
+	attack_sound = 'sound/magic/mm_hit.ogg'
+	icon = 'icons/Fulpicons/64x64.dmi'
+	icon_state = "angel"
+	icon_living = "angel"
+	health_doll_icon = "angel"
+	faction = list("apostle")
+	friendly_verb_continuous = "stares down"
+	friendly_verb_simple = "stare down"
+	speak_emote = list("proclaims")
+	armour_penetration = 100
```

Comment:
Doesnt this seem too unfair for people who bothered to get armor? I believe the only other thing that penetrates armor fully is the staff of storms

Reply:
Changed.